html/changelog.html

```diff
@@ -24,27 +24,28 @@
 <table align='center' class="top">
 	<tr>
 		<td valign='top'>
-			<div align='center'><font size='3'><b>Welcome to Colonial Marines!</b></font></div><br>
-			<div align='center'><font size='2'><i>Colonial Marines is a medium-roleplay Space Station 13 server inspired by the Aliens film and built from a heavily modified Baystation 12.</i></font></div>
+			<div align='center'><font size='3'><b>Welcome to DM;CA!</b></font></div><br>
+			<div align='center'><font size='2'><i>Digital Marines ; Colonel Alien is a low-medium roleplay Space Station 13 server branched off CM-SS13, which branched off Baystation 12, which branched off /tg/Station 13.</i></font></div>
 			<br>
-			<div align='center'><font size='3'><a href="https://cm-ss13.com/viewforum.php?f=82">Rules</a> | <a href="https://cm-ss13.com/">Forum</a> | <a href="https://cm-ss13.com/wiki">Wiki</a><br></font></div>

```

Comment:
I think we're using https://tgstation13.org/wiki/DMCA_Rules instead of the regular rules and the Addendum

Reply:
We are mainly following /tg/station ruleset, the addendum url only contain a few indications specific to this server, with no citation or direct link to the other rules. Better keep this safe and square.

Reply:
understandable! Thanks!